DEV.md

```diff
@@ -0,0 +1,232 @@
++ Top Level Design
+
+The Eclipse Transformer comprises:
+
+  * File update actions;
```

Comment:
The top level bullets should not have leading spaces. Just put the `*` at the start of the line.

Reply:
Corrected by refresh of the files.